GeneralStateTests/stSolidityTest/AmbiguousMethod.json

```diff
@@ -29,7 +29,7 @@
             ],
             "Homestead" : [
                 {
-                    "hash" : "0xc55c94f5708aff56edbe823e1e448d4e7db3d9e4c693044f633c76395219a532",
+                    "hash" : "10b11aa9b00f25bdd0a277beba65d7e74d0085af9b83ff08dab6d5304abc8e5e",
```

Comment:
why the hash is not starting with "0x" so that it can keep the same form

Reply:
#18

Reply:
> why the hash is not starting with "0x" so that it can keep the same form

I will fix it later together.